### PR TITLE
DEVTOOLS-85: Add changelog file

### PR DESCRIPTION
**Pull request by juancoen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [0.0.5] - 2015-07
+### Added
+- New create command to generate a new API in Anypoint Platform
+  - Create command can be run in interactive mode
+  - Create command can add versions to existing APIs
+- New detailed help option
+  - Detailed help is shown when tool is executed with -h parameter
```
Comment (fschargorosdky) [inline CHANGELOG.md]: - Detailed help is shown when tool is executed with -help parameter

Comment (fschargorosdky) [inline CHANGELOG.md]: - Detailed help is shown when tool is executed with --help parameter

Comment (juancoen) [inline CHANGELOG.md]: done

**Comment by fschargorosdky** [comment]
Looks good to me

**Pull request merged**